.buildkite/pipeline-release.yml

```diff
@@ -19,6 +19,14 @@ steps:
         if: build.message =~ /^buildkite test .*release.*/
         command: buildkite-agent pipeline upload .buildkite/pipeline-release-helm.yml
 
+      - label: "copy images to dockerhub"
+        if: build.tag =~ /^v[0-9]+\.[0-9]+\.[0-9]+\$/
+        trigger: unified-release-copy-elastic-images-to-dockerhub
+        build:
+          env:
+            NAMES: "eck/eck-operator,eck/eck-operator-fips,eck/eck-operator-ubi8,eck/eck-operator-ubi8-fips"
+            TAG: "${BUILDKITE_TAG}"
```

Comment:
While looking for the actual implementation it seems that the `copy.sh` script expects the following variables:

```
# Required environment variables:
#
: "$IMAGES_NAMES" # comma-separated list of the namespaced names of the Elastic images to copy
: "$IMG_TAG"      # tag of the Elastic images to copy
```

Just wanted to double check the variable names:
* `NAMES ` vs `IMAGES_NAMES`
* `TAG` vs `IMG_TAG`

Reply:
4954ba1